server/src/main/java/org/opensearch/index/search/stats/SearchStats.java

```diff
@@ -166,6 +166,10 @@ public static class Stats implements Writeable, ToXContentFragment {
 
         private long searchIdleReactivateCount;
 
+        private long starTreeQueryCount;
+        private long starTreeQueryTimeInMillis;
+        public long starTreeQueryCurrent;
```

Comment:
typo ?

Reply:
yeah - thanks for catching it. made it private as intended.